src/DataFramesMeta.jl

```diff
@@ -14,6 +14,7 @@ export @with,
        @orderby,
        @by, @combine,
        @transform, @select, @transform!, @select!,
+       @rtransform,
```

Comment:
I assume also `rtransform!`, `rselect`, `rselect!`, `rsubset`, and `rsubset!` - right?

Reply:
Yes, added as exports.